addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -243,4 +263,36 @@ img.o_forum_avatar_big {
     #o_wforum_forums_index_list & {
         min-height: 100px;
     }
+
+    & img {
```

Comment:
Again an useless `&`. In fact useless rule, add `w-100 h-100 o_object_fit_cover` in XML

Reply:
The img is created with a t-field, so I don't have access to the class attribute in the xml. There is maybe another way to do it ?

Reply:
the `classes` option as a `t-options`